src/mono/mono.proj

```diff
@@ -33,7 +33,7 @@
         <_MonoExtraCXXFLAGS>-O2 -g</_MonoExtraCXXFLAGS>
     </PropertyGroup>
     <PropertyGroup>
-        <_MonoConfigureParams Condition="'$(MonoEnableLLVM)' == 'true'">$(_MonoConfigureParams) --enable--llvm --with-llvm=$(MonoLLVMDir) </_MonoConfigureParams>
+        <_MonoConfigureParams Condition="'$(MonoLLVMDir)' != ''">$(_MonoConfigureParams) --with-llvm=$(MonoLLVMDir) </_MonoConfigureParams>
```

Comment:
the more canonical way to do this would be:

```diff
-<MonoLLVMDir Condition="'$(MonoEnableLLVM)' == 'true'">$(MonoObjDir)llvm</MonoLLVMDir>
+<MonoLLVMDir Condition="'$(MonoEnableLLVM)' == 'true' and '$(MonoLLVMDir)' == ''">$(MonoObjDir)llvm</MonoLLVMDir>
```

Reply:
ah sorry, scratch that, I missed the part of your PR description where you say `--enable--llvm` is invalid :)

Reply:
Done.